packages/terraform-modules-public/aws/testnet/variables.tf

```diff
@@ -35,6 +35,37 @@ variable instance_types {
   }
 }
 
+variable iam_instance_profiles {
+  description = "Optional IAM instances profile names for each component. For example, a profile that gives CloudWatch agent permissions"
```

Comment:
Makes sense to add here the link to https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/create-iam-roles-for-cloudwatch-agent.html as some context/example for configuring this?

Reply:
Good idea. https://github.com/celo-org/celo-monorepo/pull/4542/commits/efa5916b83fca15ad05b2d909eda6fb65e6753ea